app/assets/javascripts/pageflow/editor/collections/page_links_collection.js

```diff
@@ -0,0 +1,78 @@
+pageflow.PageLinksCollection = Backbone.Collection.extend({
+  model: pageflow.PageLink,
+
+  initialize: function(models, options) {
+    this.configuration = options.configuration;
+    this.page = options.configuration.page;
+
+    this.load();
+
+    this.listenTo(this, 'add remove change', this.save);
+    this.listenTo(this.configuration, 'change:page_links', this.load);
+  },
+
+  addLink: function(targetPageId) {
+    this.addWithPosition(this.defaultPosition(), targetPageId);
+  },
+
+  canAddLink: function(targetPageId) {
+    return true;
+  },
+
+  updateLink: function(link, targetPageId) {
+    link.set('target_page_id', targetPageId);
+  },
+
+  removeLink: function(link) {
+    this.remove(link);
+  },
+
+  addWithPosition: function(position, targetPageId) {
+    this.add(this.pageLinkAttributes(position, targetPageId));
+  },
+
+  removeByPosition: function(position) {
+    this.remove(this.findByPosition(position));
+  },
+
+  findByPosition: function(position) {
+    return this.findWhere({position: position});
+  },
+
+  load: function() {
+    this.set(this.pageLinksAttributes());
+  },
+
+  save: function(referenceKey, pageId) {
```

Comment:
'pageId' is defined but never used.<br>'referenceKey' is defined but never used.